src/biocframe/BiocFrame.py

```diff
@@ -640,6 +640,16 @@ def column(self, column: Union[str, int]) -> Any:
         )
         return self.get_column(column)
 
+    def has_row(self, name: str) -> bool:
+        """
+        Args:
+            name: Name of the row.
+
+        Returns:
+            Whether a row with the specified ``name`` exists in this object.
+        """
+        return name in self.row_names
```

Comment:
This needs a check if `self.row_names` is None or not, otherwise python throws an error for "row1" in None.